pkg/cluster/spec/grafana.go

```diff
@@ -223,6 +229,49 @@ func (i *GrafanaInstance) initDashboards(e executor.Executor, spec GrafanaSpec,
 	return nil
 }
 
+// installDashboards is used by dm cluster
+func (i *GrafanaInstance) installDashboards(e executor.Executor, deployDir, clusterName, clusterVersion string) error {
```

Comment:
Could you please add some comments about why `installDashboards` is only used by dm. BTW, if this install is only used by DM, should we rename it into `installDMDashboards()` or something similar?